scripts/deploy-mainnet-5.ts

```diff
@@ -0,0 +1,112 @@
+import hardhat from "hardhat";
+const { waffle, ethers } = hardhat;
+const { deployContract, provider } = waffle;
+const BN = ethers.BigNumber;
+import { config as dotenv_config } from "dotenv";
+dotenv_config();
+const deployer = new ethers.Wallet(JSON.parse(process.env.RINKEBY_ACCOUNTS || '[]')[0], provider);
```

Comment:
```suggestion
const deployer = new ethers.Wallet(JSON.parse(process.env.MAINNET_ACCOUNTS || '[]')[0], provider);
```

I notice we're using `process.env.RINKEBY_ACCOUNTS` for all the deploy-mainnet and deploy-polygon scripts

Reply:
All our mainnet scripts have  process.env.RINKEBY_ACCOUNTS. Because of Andrew deploys those, I did not want to change it :).